docs/website/docs/walkthroughs/deploy_a_pipeline/deploy-with-modal-snippets.py

```diff
@@ -0,0 +1,57 @@
+import os
+from tests.pipeline.utils import assert_load_info
+import modal
+
+# @@@DLT_SNIPPET_START modal_image
+# Define the Modal Image
+image = modal.Image.debian_slim().pip_install(
+    "dlt>=1.1.0",
+    "dlt[duckdb]",  # destination
+    "dlt[sql_database]",  # source (postgres)
```

Comment:
nitpick, but technically mysql in this example ( i think that was typo, sorry!)

Reply:
updated